src/components/SingleSubComment.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './SingleComment.scss';
+import { IonItem, IonAvatar, IonLabel, IonText } from '@ionic/react';
+
+interface CommentProps {
+    comment: Array<string>;
```

Comment:
This has to be a comment object

Reply:
Hmm, I don't quite know how to work with comment object.

Reply:
Maybe @lemohammed let me rebase my work on @Special-Sharpie 's View work and then I can combine everything together.

Reply:
We'll merge everything in together that way